### PR TITLE
Add api promotion rooms

### PR DESCRIPTION
**Pull request by MariethPerez**
Add api promotion rooms
promotion rooms rspec

### CODE REVIEW

**Comment by MariethPerez** [comment]
Resolve #25

File: app/controllers/api/promotions_rooms_controller.rb [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+      render json: { message: e.message }, status: :not_found
+    end
+    
+    private    
+    def set_promotion
+      @promotion = Promotion.find(params[:id])
+    end
+    
+    def promotion_params
+      params.permit(:start_date, :end_date, :discount_type, :discount_amount, :promotionable_type, :promotionable_id)
```
Comment (paulotijero) [inline app/controllers/api/promotions_rooms_controller.rb]: I think not necessary these params:
```ruby
params.permit(... , :promotionable_type, :promotionable_id)
```

File: app/controllers/api/promotions_rooms_controller.rb [hunk 2]
```diff
@@ -0,0 +1,48 @@
+class Api::PromotionsRoomsController < ApiController
+    before_action :set_promotion, only: [:show, :update, :destroy]
```
Comment (condef5) [inline app/controllers/api/promotions_rooms_controller.rb]: Hey, fixed the identation :)

File: spec/controllers/api/promotions_rooms_controller_spec.rb [hunk 3]
```diff
@@ -0,0 +1,131 @@
+require 'rails_helper'
+
+RSpec.describe Api::PromotionsRoomsController, type: :controller do
+    before do
+        @hotel = Hotel.create(
```
Comment (condef5) [inline spec/controllers/api/promotions_rooms_controller_spec.rb]: Please fix idenation here too.

File: app/controllers/api/promotions_rooms_controller.rb (hunk 1 above)
Comment (MariethPerez) [reply app/controllers/api/promotions_rooms_controller.rb]: I will fix in the next pull request.

**Pull request merged**